client/src/components/Feed/utils.js

```diff
@@ -56,3 +56,35 @@ export const setQueryKeysValue = (history, newQuery) => {
     search: stringifiedQuery,
   });
 };
+
+export const getPostedTime = (post) => {
+  const today = new Date();
+  const postCreatedAt = new Date(post.createdAt);
+  const splitDate = (date) => {
+    const day = date.getDate();
+    const month = date.getMonth() + 1;
+    const year = date.getFullYear();
+    const hour = date.getHours();
+    return {
+      year,
+      month,
+      day,
+      hour,
+    };
+  };
+
+  if (splitDate(today).year > splitDate(postCreatedAt).year)
+    return `${
+      splitDate(today).year - splitDate(postCreatedAt).year
+    } year(s) ago`;
+  if (splitDate(today).month > splitDate(postCreatedAt).month)
+    return `${
+      splitDate(today).month - splitDate(postCreatedAt).month
+    } month(s) ago`;
+  if (splitDate(today).day > splitDate(postCreatedAt).day)
+    return `${splitDate(today).day - splitDate(postCreatedAt).day} day(s) ago`;
+  if (splitDate(today).hour > splitDate(postCreatedAt).hour)
+    return `${
+      splitDate(today).hour - splitDate(postCreatedAt).hour
+    } hour(s) ago`;
```

Comment:
Shouldn't this be localized?

Reply:
This util is for Internal dashboard use. (Part of the main PR)